packages/cape-verde/README.md

```diff
@@ -0,0 +1,50 @@
+# 🇨🇻 Cape Verde
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+Contains all the provinces of Cape Verde:
```

Comment:
It looks like they are called "municipalities" (source: https://en.wikipedia.org/wiki/Administrative_divisions_of_Cape_Verde)